app/routes/auth_routes.py

```
@@ -88,51 +88,126 @@ def logout():
+#             )
+#         else:
+#             flash("Invalid user type", "danger")
+#             return redirect('/')
+
+#         db.session.add(new_user)
+#         db.session.commit()
+#         flash("Registration successful! Please log in.", "success")
+#         return redirect(f'/auth/{user_type}')
+
+#     return render_template(f'/auth/register_{user_type}.html')
+import random
```

Ruff (0.8.2)

Redefinition of unused `random` from line 2

Remove definition: `random`

(F811)